tests/s3/test_object_tagging.py

```diff
@@ -1420,6 +1352,7 @@ def test_delobj_tagging_42780(self):
         assert_utils.assert_equal(len(resp[1]), S3_OBJ_TST["test_9415"]["tag_length"], resp[1])
         self.log.info("Verify already deleted tag using DELETE object tagging")
 
+    @pytest.mark.parallel
```

Comment:
Let's not introduce this marker, unless you have verified parallel execution of the tests

Reply:
Removed.